rootfs/api/models/build.py

```diff
@@ -27,17 +27,28 @@ class Meta:
         ordering = ['-created']
         unique_together = (('app', 'uuid'),)
 
+    @property
+    def type(self):
+        """Figures out what kind of build type is being deal it with"""
+        if self.dockerfile:
+            return 'dockerfile'
+        elif self.sha:
+            return 'buildpack'
+        else:
+            # docker image (or any sort of image) used via deis pull
+            return 'image'
+
+    @property
+    def version(self):
+        return 'git-{}'.format(self.sha) if self.type == 'buildpack' else 'latest'
+
```

Comment:
hmm, yeah I guess they technically could - I'll have a bit of a look see. Was mostly basing it off the function we had in the utils file